src/opendr/engine/target.py

```diff
@@ -351,30 +366,30 @@ def __init__(
     def mot(self, with_confidence=True, frame=-1):
 
         if with_confidence:
-            result = np.array([
-                frame,
-                self.left,
-                self.top,
-                self.width,
-                self.height,
-                self.confidence,
-            ], dtype=np.float32)
+            result = np.array(
+                [
+                    self.frame,
+                    self.left,
+                    self.top,
+                    self.width,
+                    self.height,
+                    self.confidence,
+                ],
+                dtype=np.float32,
+            )
         else:
-            result = np.array([
-                frame,
-                self.left,
-                self.top,
-                self.width,
-                self.height,
-            ], dtype=np.float32)
+            result = np.array(
+                [self.frame, self.left, self.top, self.width, self.height],
```

Comment:
`self.frame` is an unresolved reference for this class. Probably meant to use just `frame` from the `mot()` method argument.